quarkus/pom.xml

```diff
@@ -71,8 +50,8 @@
         <maven.compiler.source>11</maven.compiler.source>
         <maven.compiler.target>11</maven.compiler.target>
 
-        <!-- Needs to be aligned with Quarkus, see e.g. https://github.com/quarkusio/quarkus-quickstarts/blob/2.7.6.Final/getting-started-async/pom.xml#L14 -->
-        <surefire-plugin.version>3.0.0-M5</surefire-plugin.version>
+        <!-- Needs to be aligned with Quarkus, see e.g. https://github.com/quarkusio/quarkus-quickstarts/blob/2.13.5.Final/getting-started-async/pom.xml#L14 -->
+        <surefire-plugin.version>3.0.0-M7</surefire-plugin.version>
```

Comment:
Also I noticed that the surefire plugin version needs to be aligned as well, so I just bumped the version.

Reply:
Here af9be9f I changed it in order to be able to manage even surefire version directly from the parent POM - it looks a bit nicer, especially with regards of what we're trying to achieve in general.